src/pages/collection/CollectionPage.tsx

```diff
@@ -123,19 +122,17 @@ function CollectionPage() {
 
   const Cell = columns => ({ columnIndex, key, rowIndex, style }) => {
     const index = rowIndex * columns + columnIndex;
-    const item = get(items, `${index}`, null);
-    if (item === null) {
-      const neededPage = Math.ceil((index + 1) / pageSize);
-      if (fetchedPages.indexOf(neededPage) === -1) {
-        fetchPage(neededPage);
-        return (
-          <div key={key} style={style}>
-            {renderPlaceholder()}
-          </div>
-        );
-      }
-      return null;
+    const neededPage = Math.ceil((index + 1) / pageSize);
+    const groupList = fetchedPages[neededPage];
+    if (groupList == undefined) {
```

Comment:
Before somebody asks, this is on purpose so it will check for both `undefined` and `null`.